xml/bp_troubleshooting_tips.xml

```diff
@@ -744,20 +641,25 @@ ID WEIGHT TYPE NAME    UP/DOWN REWEIGHT PRIMARY-AFFINITY
    As a general rule, time synchronization must be configured and running on
    all nodes. Once time synchronization is set up, the clocks should not get
    skewed. However, if a clock skew is to occur, this is likely due to the
-   <systemitem class="daemon">chronyd.service</systemitem> not running on one or more hosts.
+
+   <systemitem class="daemon">chronyd.service</systemitem> not running on one
+   or more hosts.
   </para>
 
   <note>
-    <para>
-      It is also possible that the battery on the motherboard has died, and
-      the clock skew will be more pronounced. If this is the case, be aware
-      that it will take quite some time for chronyd to re-sync the clocks.
-    </para>
+   <para>
+    It is also possible that the battery on the motherboard has died, and the
+    clock skew will be more pronounced. If this is the case, be aware that it
+    will take quite some time for chronyd to re-sync the clocks.
+   </para>
   </note>
+
   <para>
-    If you receive a clock skew warning, confirm that the <systemitem class="daemon">chronyd.service</systemitem>
-    is running on all cluster nodes. If not, restart the service and wait for
-    chronyd to re-sync the clock.
+   If you receive a clock skew warning, confirm that the
+   <systemitem class="daemon">chronyd.service</systemitem> is running on all
```

Comment:
```suggestion
   <systemitem class="daemon">chronyd.service</systemitem> daemon is running on all
```

Reply:
Or "service", but since we introduce it with "the" we need some classification here